### ISSUE TITLE
Support debug_url in the manifest

### ISSUE THREAD

**Issue by Hacksore** [AlbericoD/overwolf-modern-react-boilerplate#3]
**Is your feature request related to a problem? Please describe.**
Currently, the focus of the project seems to be, develop in the browser and build for overwolf later.

**Describe the solution you'd like**
We should support the `debug_url` property in the manifest to allow for react hot reloading as well is reactive development within the overwolf client.

**Describe alternatives you've considered**
N/A

**Additional context**
tl;dr this is great for web browser base development but we should instead shift to allowing for development while in-game using the overwolf client.
